src/com/lftechnology/java/training/alina/jdbc/controller/EmployeeController.java

```diff
@@ -0,0 +1,236 @@
+            } else {
+                LOGGER.log(Level.INFO, "\n=====>\nFailed to add new employee.\n=====>\n");
+            }
+        }
+    }
+
+    /**
+     * Check if the role is either admin/user
+     * 
+     * @param scanner
+     *            {@link Scanner}
+     * @param roleStatus
+     *            {@link Boolean}
+     * @param role
+     *            {@link String}
+     * @return roleStatus {@link Boolean} role status as admin/user
+     * @author Alina Shakya <alinashakya@lftechnology.com>
+     */
+    public static boolean checkMatchedRole(Scanner scanner, Boolean roleStatus, String role) {
+        if (role.equals(EmployeeRole.ADMIN.role) || role.equals(EmployeeRole.USER.role)) {
+            roleStatus = true;
+        } else {
+            LOGGER.log(Level.INFO, "\n=====>\nRole should be user/admin. Please enter valid role\n=====>\n");
+            roleStatus = false;
+        }
+        return roleStatus;
+    }
+
+    /**
+     * Delete existing employee by username
+     * 
+     * @param scanner
+     *            {@link Scanner}
+     * @author Alina Shakya <alinashakya@lftechnology.com>
+     */
+    public static void deleteExistingEmployee(Scanner scanner) {
+        Employee employee = new Employee();
+        employee.setFullname(UtilityService.getInputData(scanner, "Enter fullname to delete : "));
+        boolean isDeleted = employeeDao.delete(employee.getFullname());
+        if (isDeleted) {
+            LOGGER.log(Level.INFO, "\n=====>\nSuccessfully deleted employee with Name {0}.\n=====>\n",
+                    new Object[] { employee.getFullname() });
+        } else {
+            LOGGER.log(Level.WARNING, "\n=====>\nFailed to delete an employee with Name {0}.\n=====>\n",
+                    new Object[] { employee.getFullname() });
+        }
+    }
+
+    /**
+     * Terminate existing employee
+     * 
+     * @param scanner
+     *            {@link Scanner}
+     * @author Alina Shakya <alinashakya@lftechnology.com>
+     */
+    public static void terminateExistingEmployee(Scanner scanner) {
+        Employee employee = new Employee();
+        employee.setFullname(UtilityService.getInputData(scanner, "Enter fullname : "));
+        boolean isDeleted = userDao.delete(employee.getFullname());
+        if (isDeleted) {
+            LOGGER.log(Level.INFO, "\n=====>\nSuccessfully terminated employee with Name {0}.\n=====>\n",
+                    new Object[] { employee.getFullname() });
+        } else {
+            LOGGER.log(Level.WARNING, "\n=====>\nFailed to terminate an employee with Name {0}.\n=====>\n",
+                    new Object[] { employee.getFullname() });
+        }
+    }
+
+    /**
+     * Gets Employee lists soreted by alphabetical order
+     * 
+     * @author Alina Shakya <alinashakya@lftechnology.com>
+     */
+    public static void getEmployeeList() {
+        List<Employee> list = employeeDao.findAll();
+        Collections.sort(list);
+        if (list.size() > 0) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Use isEmpty() to check whether the collection is empty or not. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1155)